cmd/telegraf/telegraf.go

```diff
@@ -223,11 +229,6 @@ func (t *Telegraf) watchLocalConfig(ctx context.Context, signals chan os.Signal,
 			log.Printf("I! Config file %q overwritten\n", fConfig)
 		} else {
 			log.Printf("W! Config file %q deleted\n", fConfig)
```

Comment:
@srebhan you may not be able to see it but after seeing a Deleted message we wait for a full second to determine if the file came back as part of a replace, or if it was actually deleted.

Do you want to extend this time even longer?

Reply:
No I'm fine with this and I probably simply missed the code...